pkg/openstack/dataplane.go

```diff
@@ -62,7 +62,7 @@ func DataplaneNodesetsDeployed(version *corev1beta1.OpenStackVersion, dataplaneN
 		if !nodeset.IsReady() {
 			return false
 		}
-		if nodeset.Status.DeployedVersion != version.Spec.TargetVersion {
+		if !nodeset.Status.Conditions.IsTrue(dataplanev1.NodeSetMinorUpdateReadyCondition) {
```

Comment:
The linter is complaining now because we don't use the `version *corev1beta1.OpenStackVersion` that is the first arg to the function.  We can remove it from the functions args if it's not needed any more.

Reply:
Fixed.

Reply:
This code isn't changed in the latest revision.